src/importSelect/ImportSelect.tsx

```diff
@@ -0,0 +1,189 @@
+  testID?: string
+}) {
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.SoftLight} testID={testID}>
+      <Touchable borderRadius={8} style={styles.touchable} onPress={onPress}>
+        <View style={styles.cardContent}>
+          <View style={styles.topLine}>
+            {icon}
+            <Text style={styles.cardTitle}>{title}</Text>
+          </View>
+          <Text style={styles.cardDescription}>{description}</Text>
+        </View>
+      </Touchable>
+    </Card>
+  )
+}
+
+export default function ImportSelect({ navigation }: Props) {
+  const dispatch = useDispatch()
+  const headerHeight = useHeaderHeight()
+  const { t } = useTranslation()
+  const onboardingProps = useTypedSelector(onboardingPropsSelector)
+
+  const handleNavigateBack = () => {
+    dispatch(cancelCreateOrRestoreAccount())
+    ValoraAnalytics.track(OnboardingEvents.restore_account_cancel)
+    navigateClearingStack(Screens.Welcome)
+  }
+
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerLeft: () => (
+        <TopBarTextButtonOnboarding
+          title={t('cancel')}
+          onPress={handleNavigateBack}
+          titleStyle={{ color: colors.gray5 }}
+        />
+      ),
+      headerStyle: {
+        backgroundColor: 'transparent',
+      },
+    })
+  }, [navigation])
+
+  return (
+    <SafeAreaView style={styles.safeArea} edges={['bottom']}>
+      <ScrollView
+        contentContainerStyle={styles.scrollContainer}
+        style={[headerHeight ? { marginTop: headerHeight } : undefined]}
+      >
+        <View style={styles.viewContainer}>
+          <View style={styles.screenTextContainer}>
+            <Text style={styles.screenTitle}>{t('importSelect.title')}</Text>
+            <Text style={styles.screenDescription}>{t('importSelect.description')}</Text>
+          </View>
+          <ActionCard
+            title={t('importSelect.emailAndPhone.title')}
+            description={t('importSelect.emailAndPhone.description')}
+            icon={<CloudCheck />}
+            onPress={() => {
+              goToNextOnboardingScreen({
+                firstScreenInCurrentStep: Screens.ImportSelect,
+                onboardingProps: { ...onboardingProps, chosenRestoreType: ChosenRestoreType.Cloud },
+              })
+            }}
+            testID="ImportSelect/CloudBackup"
+          />
+          <ActionCard
+            title={t('importSelect.recoveryPhrase.title')}
+            description={t('importSelect.recoveryPhrase.description')}
+            icon={<Lock />}
+            onPress={() => {
+              goToNextOnboardingScreen({
+                firstScreenInCurrentStep: Screens.ImportSelect,
+                onboardingProps: {
+                  ...onboardingProps,
+                  chosenRestoreType: ChosenRestoreType.Mnemonic,
+                },
+              })
+            }}
+            testID="ImportSelect/Mnemonic"
+          />
+        </View>
+      </ScrollView>
```

Comment:
You are missing the 'Create a new wallet' button https://www.figma.com/file/Qj4kphntYzlYwex79XTcEj/Wallet-Recovery-1.5?node-id=3604%3A33922&mode=dev

Reply:
Part of the AC for the ticket specifies it should not be included: [ACT-874](https://linear.app/valora/issue/ACT-874/restore-options-screen-emailphone-or-manual).